packages/graphql-explorer-html/src/ConnectionParamsPanel.tsx

```diff
@@ -0,0 +1,39 @@
+import { JsonInput, jsonField } from 'graphql-explorer/lib/JsonInput';
+import React from 'react';
+import Form from 'react-formal';
+import * as yup from 'yup';
+
+const schema = yup.object({
+  uri: yup
+    .string()
+    .default('https://api.graph.cool/simple/v1/swapi')
```

Comment:
if we have the uri defaulted by react state anyway, why default it here? maybe I misunderstand what this is doing

Reply:
when you open the app, the state would be empty. so you get this default